go/vt/topo/etcd2topo/lock.go

```diff
@@ -17,26 +17,38 @@ limitations under the License.
 package etcd2topo
 
 import (
-	"flag"
+	"context"
 	"fmt"
 	"path"
 
-	"context"
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
 
 	"go.etcd.io/etcd/api/v3/mvccpb"
 	clientv3 "go.etcd.io/etcd/client/v3"
 
-	"vitess.io/vitess/go/vt/proto/vtrpc"
-	"vitess.io/vitess/go/vt/vterrors"
-
 	"vitess.io/vitess/go/vt/log"
+	"vitess.io/vitess/go/vt/proto/vtrpc"
 	"vitess.io/vitess/go/vt/topo"
+	"vitess.io/vitess/go/vt/vterrors"
 )
 
 var (
-	leaseTTL = flag.Int("topo_etcd_lease_ttl", 30, "Lease TTL for locks and leader election. The client will use KeepAlive to keep the lease going.")
+	leaseTTL = 30
 )
 
+func init() {
+	for _, cmd := range []string{"vttablet", "vtctl", "vtctld", "mysqlctl", "mysqlctld", "vttestserver", "vtcombo", "vtctldclient", "vtexplain", "vtgate",
```

Comment:
I have to update this list.

Reply:
This list is correct. All of these binaries pull in topo server implementations and use those to connect to the actual etcd server, so the flags are required for all of them.

Reply:
actually... mysqlctl and mysqlctld probably don't need these flags.
we can clean that up later.